debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
Yes, the version is picked up from debian/changelog. You can look at tools like `dch` for programmatically manipulating this. The rules on version numbers are described [here](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version).